flutter/lib/src/integrations/web_sdk_integration.dart

```diff
@@ -1,32 +1,48 @@
 import 'dart:async';
 
 import 'package:meta/meta.dart';
+import 'package:sentry/sentry.dart';
 
-import '../../sentry_flutter.dart';
+import '../native/sentry_native_binding.dart';
+import '../sentry_flutter_options.dart';
 import '../web/script_loader/sentry_script_loader.dart';
 import '../web/sentry_js_bundle.dart';
 
+Integration<SentryFlutterOptions> createSdkIntegration(
```

Comment:
where is this used?
I didn't find a reference to it in the code. Is it coming in another PR?

Reply:
it's used within `sentry_flutter.dart`

`integrations.add(createSdkIntegration(native));`

Reply:
but there is also the same `Integration<SentryFlutterOptions> createSdkIntegration` in `native_sdk_integration.dart`, and `sentry_flutter.dart` seems to use that one

Reply:
ah yes, we are using conditional imports (see `sdk_integration.dart`, we can define the function and implement it for either native platforms or web

if you're using the IDE I guess it's not really showing the usage correctly

Reply:
> got it, i think, in the sdk_integration.dart exports
there's no clashing issue between native_sdk_integration.dart and web_sdk_integration.dart, right?

yes they dont clash

we only compile web_sdk_integration if there is either the `dart:html`or `dart:js_interop` libraries available and those are only available when compiling in web, otherwise we just use the normal native integration